cmake/setup_custom_targets.cmake

```diff
@@ -121,7 +120,6 @@ FILE(WRITE ${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/print_info.cmake
 #    documentation  - ${_description_string} component 'documentation'
 #    examples       - ${_description_string} component 'examples'
 #    library        - ${_description_string} component 'library'
-#    mesh_converter - ${_description_string} component 'mesh_converter'
 #    parameter_gui  - ${_description_string} component 'parameter_gui'
 #    package        - build binary package
```

Comment:
Great! And with the *GridOut::write_ucd* function, the overall ability to translate meshes is not lost!